adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -282,7 +289,7 @@ private void handleEndpointConnectionWithAuthentication(final MqttEndpoint endpo
     void handleCredentialsResult(final AsyncResult<String> attempt, final MqttEndpoint endpoint, final UsernamePasswordCredentials authObject) {
         if (attempt.succeeded()) {
             String logicalDeviceId = attempt.result();
-            LOG.trace("successfully authenticated device id <{}>", logicalDeviceId);
+            LOG.info("successfully authenticated device id [{}]", logicalDeviceId);
```

Comment:
should remain at trace level FMPOV

Reply:
it should be the counterpart of the "successfully connected..." in the handle without authentication so at least debug fmpov.